services/common/workflow.js

```diff
@@ -1,7 +1,15 @@
-import { TYPE } from "./changeRequest.js";
-
 export const ONEMAC_TYPE = {
-  ...TYPE,
+  CHIP_SPA: "chipspa",
```

Comment:
Yes, somehow the config files in the common have gotten all tangled and so I can't use the constants I need in the places I want at this time, but we should spend a little time reorganizing so that those constants can continue to be used :)   Another TODO!